maat-court-data-api/src/main/java/gov/uk/courtdata/dces/util/ContributionFileUtil.java

```diff
@@ -67,6 +105,60 @@ private ContributionFileErrorsEntity buildBaseErrorEntity(LogProcessedRequest re
                 .build();
     }
 
+    public void setPreparedStatementParameters(PreparedStatement ps, Map<String,String> fieldValueMap, ContributionFilesEntity contributionFilesEntity, Clob xmlContent, Clob ackXmlContent, boolean isUpdate) throws SQLException {
+        int parameterIndex = 1;
+
+        if(fieldValueMap.containsKey(FILE_NAME)) {ps.setString(parameterIndex++, contributionFilesEntity.getFileName());}
```

Comment:
I've swapped to using a switch/case statement, as the map is mainly used for populating the sql, so it would complicate the map to add a consumer to it too.